lock-api/src/main/java/com/palantir/lock/client/ProfilingTimelockService.java

```diff
@@ -112,8 +113,8 @@ public LockImmutableTimestampResponse lockImmutableTimestamp() {
     }
 
     @Override
-    public StartIdentifiedAtlasDbTransactionResponse startIdentifiedAtlasDbTransaction() {
-        return runTaskTimed("startIdentifiedAtlasDbTransaction", delegate::startIdentifiedAtlasDbTransaction);
+    public List<StartIdentifiedAtlasDbTransactionResponse> startIdentifiedAtlasDbTransactionBatch(int count) {
+        return runTaskTimed("startIdentifiedAtlasDbTransactionBatch", () -> delegate.startIdentifiedAtlasDbTransactionBatch(1));
```

Comment:
You need to propagate the count?

Reply:
Good catch - autoreplaced the code and I obviously missed this.